kolibri/core/assets/src/views/nav-bar/index.vue

```diff
@@ -50,7 +50,7 @@
 
 <script>
 
-  const values = require('lodash.values');
+  const values = require('lodash/values');
```

Comment:
Upon closer inspection, i'm seeing that it's a capitalization problem.

we're requesting `differencewith` rather than `differenceWith` and `orderby` rather than `orderBy`.

Dunno why it's working on your machine, but fixing this allowed it to run on mine.